test/adapter_test.rb

```diff
@@ -8,10 +8,10 @@ def setup
       @adapter = ActiveModelSerializers::Adapter::Base.new(@serializer)
     end
 
-    def test_serializable_hash_is_abstract_method
-      assert_raises(NotImplementedError) do
-        @adapter.serializable_hash(only: [:name])
-      end
+    def test_serializable_ensures_options_is_a_hash
```

Comment:
I still find it really weird that the `serializable_hash` method would return options.